SU-SWS/decanter
# 223 brand bar

# READY FOR REVIEW
- A first pass but would like a review as a sanity check

# Summary
- Global brand bar component with 4 variants

# Needed By (Date)
- Sometime this week before Friday if possible

# Urgency
- Not urgent

# Steps to Test

1. Check out this branch.
1. Run grunt styleguide
2. Click on "Components" and look at section 3.2 "Brand Bar" and check that the 4 variants match the design in the Figma document Stanford Design Library (under "Header & Footer" -> "Brand Bar")

# Affected Projects or Products
- Future Drupal and Wordpress templates

# Associated Issues and/or People
- JIRA ticket https://github.com/SU-SWS/decanter/issues/223
- It makes use of the CSS grid mixin and is dependent of the current breakpoints
